### ISSUE TITLE
getting error when building for android.

### ISSUE THREAD

**Issue by surendra019** [EIRTeam/EIRTeam.FFmpeg#29]
after installing all the dependencies and implementing commands, it still throws an error.
![error](https://github.com/user-attachments/assets/c7ab6258-0ba3-458e-9ec8-95d2674d2de2)

**Comment by surendra019** [comment]
your android build is not working, also your windows build doesn't play video. In fact, the latest windows build provided in the official repo is also not working. But only thing worked is the 1.0  version of windows build.

**Comment by EgrorBs** [comment]
I checked, everything works for me, both on Windows 10 and Android, I have godot-stable version

**Comment by surendra019** [comment]
please help me to make it work, I am finding its solution since 2 days. i have windows 11 and godot 4.2 stable.

**Comment by EgrorBs** [comment]
You can try upgrading to godot-latest, if it doesn't break your project

**Comment by EgrorBs** [comment]
I got latest version from godot auto build, I don't remember what the last one was at that time, but for example this one should work

https://github.com/godotengine/godot/actions/runs/9898707467
(Artifacts section, at the very bottom)

**Comment by surendra019** [comment]
see this issue - 
https://drive.google.com/file/d/1ch8xJOqXyNXddSGWKz0wdWes7dLieVyE/view?usp=sharing

**Comment by EgrorBs** [comment]
I don't know if this makes sense, but to rule out accidents: I started the scene with the paused flag set and then ran the video through the script

Now I will conduct a couple more tests with my setup, and write about the results

**Comment by surendra019** [comment]
your version worked in windows. let me try in android.

**Comment by EgrorBs** [comment]
You can add template in same place you have download stable, just "Add from file"

**Comment by surendra019** [comment]
but it asks .tpz file

**Comment by EgrorBs** [comment]
I think you can just place files here 
![image](https://github.com/user-attachments/assets/c47472c9-a693-46e8-aab9-ae8a9e9e9cfc)

**Comment by EgrorBs** [comment]
[game3.zip](https://github.com/user-attachments/files/16195348/game3.zip)
For me it works in this minimal setup on Win10 and Android 11 (i used `eirteam-ffmpeg-5c249e3.zip` from my build)

**Comment by EgrorBs** [comment]
I think I found the problem: everything works on the "Compatibility" renderer, but not on "Mobile" or "Forward+" at least on Windows

**Comment by EgrorBs** [comment]
As maintainer says [here](https://github.com/EIRTeam/EIRTeam.FFmpeg/issues/23#issuecomment-2175090420), and from what I've found out, you need either version 4.3 or the "Compatibility" renderer

**Comment by EgrorBs** [comment]
Have you figured out how to install an export template?

**Comment by surendra019** [comment]
yes, but by enabling gradle build for export, it throws an error, so now i am trying the compatibility mode.

**Comment by EgrorBs** [comment]
I don't know much about building for android, but maybe you manually placed files in the 'res://android/build' folder? In theory, they are different for different versions of godot, at least in my understanding they should be

**Comment by EgrorBs** [comment]
Android templates from my 4.3 godot build [4.3.dev6.zip](https://drive.google.com/file/d/191GXpeAMnCOAMsXic544SVQS9HWqdiZU/view?usp=sharing)

**Comment by EgrorBs** [comment]
@surendra019 I make fix to force latest release of EIRTeam.FFmpeg work on any godot branch (by allowing YUV -> RGB GPU convertion only on godot 4.3+). On godot stable its more laggy (then godot latest), but it seems to work properly on any configuration (4.2.stable/4.3.latest, win/android)

https://github.com/EgrorBs/EIRTeam.FFmpeg/actions/runs/9915609753

**Comment by EIREXE** [comment]
Sorry for the confusion caused by the YUV conversion, I didn't expect godot to take so long to release 4.3 :sweat_smile:
